### DISCUSSION TITLE
How to use web3modal v2 vanilla-js?

### DISCUSSION THREAD

**Discussion by C0INON** [WalletConnect/walletconnect-monorepo#2512] [Developer Support]
The current version of v1 is used by https://github.com/WalletConnect/web3modal-vanilla-js-example in this example. I'm not using node or react, I'm using php + html + vanilla js through Apache server. But v2 doesn't have an example. Is there anyone who can help me?

**Comment by finessevanes** [comment]
Hey - this is on the roadmap. I've asked our team for an ETA.

**Comment by finessevanes** [comment]
You can follow this [issue](https://github.com/WalletConnect/web3modal/issues/678) for the progress.
